frontend/src/components/meta-title.tsx

```diff
@@ -1,3 +1,9 @@
-export function MetaTitle({ title }: { title: string }) {
-  return <title>{`${title} | ZaneOps`}</title>;
-}
+import type { FC } from "react";
+
+type MetaTitleProps = {
+  title: string;
+};
+
+export const MetaTitle: FC<MetaTitleProps> = ({ title }) => (
```

Comment:
We don't use `FC` we prefer regular functions instead, this is a deliberate choice.

Reply:
`FC` mean Functional component it's cleaner that simple function and type check can "test" it

Reply:
This is subjective, but i find writing functions cleaner, we don't really to test them with the type check, if it is used as a React node, TypeScript will check it. Please not also that React jsx nodes are very permissive, so in practice this doesn't add very big improvements.

Reply:
I already know what `FC` is and have already used it in previous projects, but functions are cleaner (for me).